lib/src/util/react_util.dart

```diff
@@ -1,14 +1,14 @@
 import 'dart:collection';
 
+import 'package:react/react_client.dart';
 import 'package:over_react/component_base.dart' as component_base show UiProps;
 import 'package:over_react/over_react.dart';
-import 'package:react/react_client.dart';
```

Comment:
Why was this import removed and then re-added in non-alphabetical order?

Reply:
accident :)